src/quic/node_quic_session.h

```diff
@@ -1437,6 +1423,15 @@ class QuicSession : public AsyncWrap,
 
   static const ngtcp2_conn_callbacks callbacks[2];
 
+  class StatsDebug {
+   public:
+    StatsDebug(QuicSession* session) : session_(session) {}
+    std::string ToString();
```

Comment:
```suggestion
    std::string ToString() const;
    
```

(This is required by https://github.com/nodejs/node/pull/31446. The blank line before `public:`, `private:` etc. is something that the linter should complain about?)